### ISSUE TITLE
Problem decrypting password

### ISSUE THREAD

**Issue by mesteryui** [open-keychain/open-keychain#2904]
## Expected Behavior

I try to de crypt a .gpg file of a password created my computer with pass creating a folder before `pass insert hkl/kbj` and when I sync that files on my phone openkeycahin don't allow decrypt of course I used the same gpg key that I have on my computer and that mustn't be 
## Steps to Reproduce (for bugs)

1. Sync the files
2. Open openkeychain view the password 
3. Then openkeychain give error OpenGPG exception
4. In computer that problem doesn't exist

## Your Environment

* Android Version: 14
* Device Model: Redmi Note 9 Pro
* OpenKeychain Version: 6.0.4
* From Google Play or F-Droid?: F-Droid

**Comment by rafaelazvdo** [comment]
What version of gpg was used to create the key pair and what version of gpg was used to encrypt the files?

**Comment by mesteryui** [comment]
For create the keys and encrypt the files I used GPG version 2.4.5 but only happens with the files in a folder

**Comment by pspoe** [comment]
@mesteryui I'm pretty sure that files in/not in a directory is not the  problem, but AEAD which is enabled by default in gpg v2.4. See [gpg failing with no error for one of my encrypted files](https://unix.stackexchange.com/questions/697579/gpg-failing-with-no-error-for-one-of-my-encrypted-files) and [disabling AEAD in gnupg](https://usercomp.com/news/1163590/disabling-aead-in-gnupg).
TL;DR: You have to fix your encryption key on your computer and re-encrypt your passwords.

I did, however, have a similar problem with a Debian gpg v2.2.40, which does not do AEAD yet. There, some newer passwords were created with ID 0000000000000000, due to changed options hidden-encrypt-to and default-recipient and successfully decrypted due to option try-secret-key (in .gnupg/gpg.conf).

Looks like the later option has no pendant in open-keychain. After re-encrypting my passwords with an explicit recipient and synching to Android decryption worked there, too.

**Comment by XBEAST1** [comment]
I Made This PGP Progressive Web App Which Is Fully Open Source And Stores PGP Keys In Local Storage It Works Flawlessly On Any Device Make Sure To Check It Out 🖤✨
Link: https://nextpgp.vercel.app
